src/components/TextInput/styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { Inter } from 'next/font/google';
+import COLORS from '../../styles/colors';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export const InputText = styled.input<{ $error: boolean }>`
+  color: ${COLORS.existingInputColor};
+  font-size: 1rem; // 16px
+  font-family: ${inter.style};
+  font-weight: 400;
+  line-height: 19.2px;
+  word-wrap: break-word;
+  padding: 10px 20px 10px 20px;
```

Comment:
I would recommend changing to use `rem` units, since the padding is dependent on the font size. If the user decides to adjust their browser font size, the padding would look off.

Reply:
Also for line-height

Reply:
Also could just do `padding: 0.625rem 1.25rem;`, which shortcuts to the equivalent form. The first parameter is vertical padding, the second is horizontal.

Reply:
changed! thank you!